### ISSUE TITLE
Only one of two features in the GeoJSON file was displayed

### ISSUE THREAD

**Issue by JeongJun-Lee** [mapbox/mapbox.js#1061]
When I try to display a GeoJSON layer by my file in the same domain as the example of https://www.mapbox.com/mapbox.js/example/v1.0.0/geojson-marker-from-url/, only one of two features in the GeoJSON file was displayed.
But, when I import the file at http://geojson.io, both were displayed normally. 

You can see the GeoJSON file which I tested at http://download.samarkandtour.org/uz_foodndrink_20150921.geojson

And I have a question. Can I parse the properties of the feature in the GeoJSON file by Mapbox.js in additional to display  a layer? I can't find the example at Mapbox homepage.

Thanks,
JJ

**Comment by tmcw** [comment]
This displays correctly in a local testing file:

<img width="459" alt="2015-09-23 at 4 55 pm" src="https://cloud.githubusercontent.com/assets/32314/10058535/e9d8ec94-6213-11e5-8ce3-d61786f8cc8f.png">

The markers are close together: are you sure you're zoomed in far enough?

> Can I parse the properties of the feature in the GeoJSON file by Mapbox.js in additional to display a layer?

Could you rephrase that to be more clear?

**Comment by JeongJun-Lee** [comment]
Very sorry, It was my mistake. I was confused the order of longitude and latitude. I found that it was opposite between the order of at http://geojson.io and Mapbox.js.

By the way, the additional question is that I am not sure if there is a parser of a geojson file in mapbox.js. I wanted to display each value of properties of a feature in the file after parsing a geojson file. When I searched the Mapbox homepage in detail, finally I found the example, https://www.mapbox.com/mapbox.js/example/v1.0.0/multiple-marker-filters/ but there is no code to load a geojson file. Anyway, when I meet a question not a issue, is there any channel to ask it?

Thanks,
JJ
